src/main/groovy/org/gradlefx/tasks/AirPackage.groovy

```diff
@@ -65,6 +69,32 @@ class AirPackage extends DefaultTask {
         showAntOutput ant.properties[ANT_OUTPUT_PROPERTY]
     }
 
+    def initInputOutputFiles() {
+        if (project.type == FlexType.air) {
```

Comment:
I thought so too. In fact, I didn't have this initially, but I noticed that when I had logging statements in here, they were being displayed for every project regardless of type (including swf, swc, and swcAir). Hence this if statement. 

Perhaps that's harmless though? Or maybe there's a better way to do this?